cypress/e2e/registration.cy.js

```diff
@@ -1,11 +1,68 @@
 /// <reference types='cypress' />
 
 describe('Student Registration page', () => {
+  let user;
+  
   before(() => {
-
+    cy.visit('/');
+    cy.task('generateUser').then(generateUser => {
+      user = generateUser;
+    });
   });
 
-  it('', () => {
-
+  it('should provite the ability to register', () => {
+    cy.get('#firstName')
+      .type(user.firstName);
+    cy.gey('#lastName')
+      .type(user.lastName);
+    cy.get('#userEmail')
+      .type(user.email);
+    cy.log(user.gender);
+    cy.contains('.coustom-control-label', user.gender)
+      .click();
+    cy.get('#userNumber')
+      .type(user.phoneNumber);
+    cy.log(user.birthDate);
+    cy.get('#dateOfBirthInput')
+      .click();
+    cy.pickDate('month-select')
+      .select(user.birth.month);
+    cy.pickDate('year-select')
+      .select(`${user.birth.year}`);
+    cy.pickDate('day')
+      .contains(user.birth.day)
+      .click();
+    cy.get('.subjects-auto-complete__value-container')
+      .type('his{enter}');
+    cy.contains('.custom-control-label', user.hobby)
+      .click();
+    cy.get('#currentAddress')
+      .type(user.address);
+    cy.get('#state')
+      .type('{drwnarrow}{enter}')
```

Comment:
The key strokes '{drwnarrow}{enter}' are with typo

Reply:
Done